packages/jest-reporters/src/getSummary.ts

```diff
@@ -114,6 +114,10 @@ export default function getSummary(
   const testsTotal = aggregatedResults.numTotalTests;
   const width = (options && options.width) || 0;
 
+  const seed = `${
+    chalk.bold('Seed:        ') + (options?.seed?.toString() ?? 'XXXX')
```

Comment:
instead of falling back to `xxxx`, we should throw an error if `options?.showSeed` but `options?.seed` is undefined